tests/src/Functional/ConfigurableLanguageTest.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types = 1);
 
-namespace Drupal\Tests\oe_multilingual\Functional;
+namespace Drupal\Tests\oe_multilingual\src\Functional;
```

Comment:
This is not a correct namespace. See everywhere, namespaces shiuld not have src in them